src/pretix/helpers/templatetags/thumb.py

```diff
@@ -46,3 +47,30 @@ def thumb(source, arg):
         # default_storage.url works for all files in NanoCDNStorage. For others, this may return an invalid URL.
         # But for a fallback, this can probably be accepted.
         return source.url if hasattr(source, 'url') else default_storage.url(str(source))
+
+
+@register.filter
+def thumbset(source, arg):
+    cache_key = f"thumbset:{source}:{arg}"
+    cached_thumbset = cache.get(cache_key)
+    if cached_thumbset is not None:
+        return cached_thumbset
+    formats = list(set().union(
+        settings.PILLOW_FORMATS_IMAGE,
+        settings.PILLOW_FORMATS_QUESTIONS_FAVICON,
+        settings.PILLOW_FORMATS_QUESTIONS_IMAGE
+    ))
+
+    srcs = []
+    if not cached_thumbset:
+        for thumbsize, factor in get_srcset_sizes(arg):
+            try:
+                t = get_thumbnail(source, thumbsize, formats=formats, only_if_resized=True)
```

Comment:
`only_if_resized=True` creates the problem, that if I e.g. upload a product image sized 180x180, which would be perfect for 3x is discarded from srcset as it is not resized. So we would need something like `only_if_smaller_or_same`?